### DISCUSSION TITLE
특정 장비에서 월드 입장이 안되는 현상

### DISCUSSION THREAD

**Discussion by YSHFM** [naverz/zepeto-studio-kor#1141] [5. Q&A]
현재 제작 중인 월드 패키지가 765mb정도 됩니다

ipadOS 15.6,
앱버전은 3.14.2
유니티 월드 패키지는 1.5.1입니다.

사용 중인 테스트 장비 중에서 아이패드 6세대만 해당 월드 입장 시 93%에서 항상 앱이 강제종료되는 문제가 있습니다.
테스트 장비 중 갤럭시와 다른 아이패드 에어 최신기종에서는 정상적으로 로드됩니다만 일부 특정 장비에서만 월드 입장이 안되는 것 같습니다.

이유 아시는 분이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 저희가 검수 시 기준으로 사용하는 최저 사양 기기는 아이폰 6, 갤럭시 S6입니다.
추후 최저 사양 기기 기준을 아이폰8 정도로 높일 계획이구요
아이패드 6세대가 2018년에 출시된 기기이다보니 최저 사양을 충족하지 않아 접속이 되지 않는 것으로 보여집니다.
최적화 및 퀄리티 세팅을 낮춰서 빌드해보시기를 추천 드립니다.
